pkg/metrics/kubelet_metrics.go

```diff
@@ -126,6 +130,22 @@ func NewKubeletMetrics() KubeletMetrics {
 	return KubeletMetrics(result)
 }
 
+// GrabKubeletMetricsWithoutProxy retrieve metrics from the kubelet on the given node using a simple GET over http.
+// Currently only used in integration tests.
```

Comment:
This function is only used in the integration test. :)
Here is the [original code](https://github.com/kubernetes/kubernetes/blob/master/test/e2e/framework/kubelet_stats.go#L405-L418).